recipe/meta.yaml

```diff
@@ -30,6 +30,8 @@ requirements:
     - giflib 5.1.*
     - freetype 2.6.*  # [linux]
     - xorg-libxaw  # [linux]
+    - zlib
+    - python  # [osx]
```

Comment:
This will probably make a matrix on Mac when rerendered; you should either add a `skip` or just make it `python 3.6` or something.

Reply:
Ah it does. That seems like a bug. It should only care about that if it is a run dependency.

Reply:
Yeah, agreed.

Reply:
I *think* this is handled already in conda-smithy 3 (https://github.com/conda-forge/conda-smithy/issues/415#issuecomment-377823810).